ddp-workspace/projects/ddp-dsm-ui/src/app/participant-list/participant-list.component.ts

```diff
@@ -2275,4 +2299,12 @@ export class ParticipantListComponent implements OnInit {
      });
      this.orderColumns();
   }
+
+  private sendAnalyticsMetric(  ) {
+    let passed = new Date().getTime() - this.start;
+    this.dsmService.sendAnalyticsMetric(this.getRealm(), passed).subscribe(
```

Comment:
The deprecated `subscribe` syntax - "Instead of passing separate callback arguments, use an observer argument." 

`.subscribe(data=>{}, err=>{}); ` => `subscribe({})`